web.py

```diff
@@ -139,5 +187,38 @@ def compile(emit, optimize, version, color, syntax):
         else:
             return {"result": split[1].decode()}
 
-os.chdir(sys.path[0])
-run(host='0.0.0.0', port=80, server='cherrypy')
+def main(args):
+    '''
+    -d: Debugging mode: turn on verbose debugging (and RUST_BACKTRACE=1)
+        Change the listening port to 8080 from the default 80.
+    -S: Skip playpen and direcly run rustc and the output binary.
+        To mitigate the risk, accepts only local connections by default.
```

Comment:
The idea is nice (especially the only local connections thing), but I'm pretty sure we don't really want this, since the behaviour under playpen has to be tested, since it differs from normal execution. For example, I'm now getting this when trying to do anything but format (when running with playpen):

```
playpen: execvpe: No such file or directory
playpen: application terminated with error code 1
```

Due to some interaction with multirust (I suspect it's because everything runs as a different user), I *also* can't get this to work without playpen:

```
error: no default toolchain configured
```